plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeColumnHandle.java

```diff
@@ -67,6 +71,8 @@
     // The value is same as 'delta.columnMapping.physicalName' when the column mapping mode is id or name. e.g. row(col-5924c8b3-04cf-4146-abb5-2c229e7ff708 integer)
     private final Type physicalType;
     private final DeltaLakeColumnType columnType;
+    private final int baseColumnIndex;
+    private final Optional<DeltaLakeColumnProjectionInfo> projectionInfo;
```

Comment:
update retained size to include `projectionInfo`

Reply:
I have used projectionInfo#partialName